applications/openshift/kubelet/kubelet_configure_tls_cipher_suites_kubeapiserver_operator/rule.yml

```diff
@@ -0,0 +1,42 @@
+documentation_complete: true
+
+prodtype: ocp4
+
+title: "Ensure that the Kubernetes API Server only makes use of Strong Cryptographic Ciphers"
+
+description: |-
+  Ensure that the Kubernetes API Server is configured to only use strong cryptographic ciphers.
```

Comment:
More accurately, this is covering the Kubernetes API Server _Operator_ TLS config. The API Server itself is covered by other rules. But for this, it is concerned with the serving endpoints of the operator.

Reply:
@mrogers950 I agree that "Operator" should be added; I updated the content (both for kube API server operator and OpenShift API server operator).